packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -341,6 +342,48 @@ const toEqualFile = () => ({
   },
 });
 
+const syncTestBodyBuilder = task => () => {
+  function runTask() {
+    run(task);
+  }
+
+  const throwMsg = task.options.throws;
+  if (throwMsg) {
+    // internal api doesn't have this option but it's best not to pollute
+    // the options object with useless options
+    delete task.options.throws;
+
+    assert.throws(runTask, function (err) {
+      assert.ok(err.message.indexOf(throwMsg) >= 0);
+      return true;
+    });
+  } else {
+    if (task.exec.code) {
+      const result = runTask;
+      if (result && typeof result.then === "function") {
+        return result;
```

Comment:
Do we need the sync and async handles, or can we handle `.rejects` here?

Reply:
I don't think we can handle `.rejects` here because `task.exec` is not defined. In #11141 it was used to reproduce the `"useSpread"` error message in test fixtures.

`assert.rejects` does behave like `assert.throws` except for the asynchronous behaviour, but I tend to be specific about the error synchrony.